### PR TITLE
Dropdown

### LINKED ISSUE #211: Dropdown menu for Cog Management
**Who are you?**
Regin Potter, CSE Dev Team member

**What feature do you want?**
I want every command (besides sync) in the Cog Management file to have a drop-down menu instead of us typing it in.

**Why do you want this feature?**
Currently we need to type in the name of the file. If we mess up the name we must re-issue the command.
This feature would be more Quality-of-Life, however I think it would be nice to implement.

**Additional context**
Some information I found was [here](https://gist.github.com/lykn/a2b68cb790d6dad8ecff75b2aa450f23).

### PR DESCRIPTION
**Pull request by ReginPotter**
# Description

I added an autofill feature to all commands in the CogManagement cog.

## Issues

Closes #211 

## Type of change

Select one or more of the following:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (describe below)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. List any edge cases you tested.
If you come up with any edge cases you didn't test while writing this PR, cancel the PR and test again.

- IMPORTANT!!! Don't forget to sync

- [x] Test A
        - Type in `/load` without pressing enter
        - Once the parameter shows up it should show all potential choices for the parameter 
- [x] Test B
        - Type in `/unload` without pressing enter
        - Once the parameter shows up it should show all potential choices for the parameter  
- [x] Test C 
        - Type in `/reload` without pressing enter
        - Once the parameter shows up it should show all potential choices for the parameter  

# Checklist:

- [X] All local commits have been pushed to remote
- [X] All changes on the base branch have been merged into this branch, either by rebase or merge
- [X] My code is [PEP-8](https://pep8.org/) compliant (excluding maximum line length, keep that to 100ish characters)
- [X] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [X] I have added [Google Docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) for all new functions/methods
- [X] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by NicoleP23 — APPROVED** [review]
Needed to give ``-sync`` a second to see it show up, but tested all three tests once it showed up and they all worked for me :+1:

**Review by L30N1DAS — CHANGES_REQUESTED** [review]
I think the Google docstrings should be updated to reflect the implemented autocomplete functionality.

I will review the PR more thoroughly after this change has been implemented (or I can be convinced that it is not necessary) to ensure that no additional undesired changes are implemented after my review.

**Comment by ReginPotter** [comment]
> I think the Google docstrings should be updated to reflect the implemented autocomplete functionality.
> 
> I will review the PR more thoroughly after this change has been implemented (or I can be convinced that it is not necessary) to ensure that no additional undesired changes are implemented after my review.

I have edited the docstrings to show the new functionality added. Please review again

File: Cogs/CogManagement.py [hunk 1]
```diff
@@ -24,15 +26,14 @@ async def load(self, interaction:discord.Interaction, cog_name:str):
...
         Args:
             cog_name (str): Name of the cog being loaded
+                cog_name will be autofilled with all of the cogs we currently have
 
         Outputs:
             Message to user informing them of what cog is being loaded, and when the action is done.
         """
 
-        # Finds the absolute path to the cog that will be loaded
         file = abspath('Cogs/' + cog_name + '.py')
```
Comment (NicoleP23) [inline Cogs/CogManagement.py:35]: Re-add this comment and line 35's comment to keep consistent comments along the functions please.

**Review by NicoleP23 — CHANGES_REQUESTED** [review]

**Review by L30N1DAS — APPROVED** [review]
Looks good. Ran into the following error twice but am unable to replicate it: 
image

**Review by Kokuko3 — COMMENTED** [review]
I like it, clip it and ship it.
